### DISCUSSION TITLE
Cutting boards slowly deteriorating (like anvils)

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#697] [Ideas]
This would be VERY slow, as in each board could be used to cut about a thousand items?

But every 200 or so, it "breaks" 1 stage and gains minor visual scratches (like an anvil) and eventually breaks after 5 stages (about a thousand).

**Comment by MythrilBagels** [comment]
Can I just say how horrible that would be for automation? You would have to routinely go through and replace cutting boards, thats not very friendly design right there.

**Reply by Aceplante** [reply]
its a THOUSAND. also what do you mean automation?

Maybe theres glass ones that break quicker and iron ones that last longer?

**Comment by vectorwing** [comment]
I'm not a fan of the Anvil's fragility as a mechanic, and it would certainly not be great for the Cutting Board either.

This would be introducing artificial maintenance which doesn't contribute to the game's enjoyment, especially considering how cheap the board already is. It's part of the reason why the Cooking Pot doesn't ask you to refill it with water between operations: it would be tedious, given this resource is so easy to get anyway.

**Reply by Aceplante** [reply]
fair enough
